rosdep/python.yaml

```
@@ -7915,6 +7915,19 @@ python3-shapely:
       packages: [shapely]
   rhel: [python3-shapely]
   ubuntu: [python3-shapely]
+python3-sila2lib-pip:
+  debian:
+    pip:
+      packages: [sila2lib]
+  fedora:
+    pip:
+      packages: [sila2lib]
+  gentoo:
+    pip:
+      packages: [sila2lib]
```

We're not supposed to use pip for gentoo. See https://github.com/ros/rosdistro/blob/master/CONTRIBUTING.md where it says "Note that pip cannot be used to install system packages on Gentoo."

However, we do encourage OpenSUSE. See https://github.com/ros/rosdistro/blob/master/REVIEW_GUIDELINES.md where it says "Ubuntu and Debian are required, Fedora, Gentoo, and openSUSE are encouraged if the package also exists."

So, we could just swap out the `gentoo` entry for `opensuse`.

```suggestion
  opensuse:
    pip:
      packages: [sila2lib]
```

---

@mabelzhang 
done and rebased